data/db-schema/src/main/resources/net/corda/db/schema/config/migration/scheduler-creation-v5.1.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+
+    <property name="now" value="now()" dbms="hsqldb"/>
+    <property name="now" value="now()" dbms="postgresql"/>
+
+    <changeSet author="R3.Corda" id="scheduler-creation-v5.1">
+
+        <createTable tableName="task_scheduler_log">
+            <column name="task_name" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="last_scheduled" type="DATETIME" defaultValueDate="${now}">
+                <constraints nullable="false"/>
+            </column>
+            <column name="scheduler_id" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <addPrimaryKey columnNames="task_name" constraintName="task_name_pk" tableName="task_scheduler_log"/>
```

Comment:
I'm not sure whether we have naming conventions for DB objects, but it might be better that PK constraints contain table names, to make sure they have uniques name (as multiple tables could have same field names). Not having unique constraint name might be a problem for some databases. Also when constraint violation references PK name, it would be clear for which table the problem happened.